src/Jackett.Common/Indexers/TVstore.cs

```diff
@@ -374,7 +366,13 @@ public class SeriesDetail
         public string HunName;
         public string EngName;
         public string imdbid;
-
+        public SeriesDetail(string[] seriesElement)
+        {
+            HunName = seriesElement[1].Split('=')[1].Trim('\'').ToLower();
+            EngName = seriesElement[2].Split('=')[1].Trim('\'').ToLower();
+            id = seriesElement[0].Split('=')[1].Trim('\'');
+            imdbid = seriesElement[7].Split('=')[1].Trim('\'');
```

Comment:
> well, we have the parsing problem here too, we won't know what field failed

You are not solving the problem...

Reply:
In the constructor a parse error will give a line number, so we will know what field failed. I'm planning on coming back to this section and doing a more thorough refactoring later (with all the TODOs)

Reply:
Well I remember testing this in the past, if you change this line to (wrong index) `imdbid = seriesElement[7].Split('=')[999].Trim('\'');` I'm pretty sure you won't see the error in line 374.

Please, don't do more changes in this PR, just fix remaining issues and merge.